### PR TITLE
Update coding-style-guide.md

### PR DESCRIPTION
**Pull request by osoykan**
Great doc! Thanks.

Small typo fix.

### CODE REVIEW

**Comment by pracucci** [comment]
@osoykan Would you mind signoff (DCO) your commit, please?

**Comment by osoykan** [comment]
Oh, missed that one, thanks. Done!

File: docs/contributing/coding-style-guide.md [hunk 1]
```diff
@@ -579,7 +579,7 @@ implicit and thus more prone to surprises.
 
 #### Clean Defer Only if Function Fails
 
-There is a way to sacrifice defer in order to properly close all on each error. Repetitions makes it easier to make error
+There is a way to sacrifice defer in order to properly close all on each error. Repetitions make it easier to make an error
```
Comment (squat) [inline docs/contributing/coding-style-guide.md:582]: hmm actually I think this reads more natural/native if we say `Repetition makes it easier...` i.e. in the singular

Comment (osoykan) [reply docs/contributing/coding-style-guide.md:582]: That sounds right 👍

**Pull request merged**